nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/test/java/org/apache/nifi/processors/mongodb/GetMongoIT.java

```diff
@@ -468,4 +469,37 @@ public void testKeepOriginalAttributes() {
     /*
      * End query read behavior tests
      */
+
+    /*
+     * Verify that behavior described in NIFI-5305 actually works.
```

Comment:
Maybe an explanation of the 'case' would be appropriate, like " Test that when you do foo, bar happens and when you do baz, foo happens, see NIFI-5303" ?  That would help maintainers.